plugins/extend-give-wp/src/admin/admin-page.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ *  The Extend GiveWP plugin option settings page.
+ *
+ * @package    spiralWebDB\ExtendGiveWP\Admin
+ *
+ * @since      1.0.0
+ *
+ * @author     Robert A. Gadon
+ *
+ * @link       http://spiralwebdb.com
+ *
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDB\ExtendGiveWP\Admin;
+
+use function spiralWebDB\ExtendGiveWP\_get_plugin_dir;
+
+add_action( 'admin_menu', __NAMESPACE__ . '\add_option_settings_page' );
+/*
+ * Add an option settings page to the plugin admin.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function add_option_settings_page() {
+	$hookname = add_submenu_page(
+		'options-general.php',  // parent slug
+		'Extend GiveWP -- Donation Form Option Settings', // page title
+		'Extend GiveWP', // menu title
+		'manage_options', // capability
+		'extend-give-wp-options', // menu slug
+		__NAMESPACE__ . '\render_option_page_template' // callback to output page content.
+	);
+
+	add_action( "load-{$hookname}", __NAMESPACE__ . '\sanitize_options' );
+}
+
+/*
+ * Render the option page template view file.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function render_option_page_template() {
+
+	if ( ! current_user_can( 'manage_options' ) ) {
+		return;
+	}
+
+	require_once _get_plugin_dir() . '/src/admin/views/option-page-template.php';
+}
+
+add_action( 'admin_init', __NAMESPACE__ . '\initialize_option_settings' );
+/*
+ * Initialize settings on the option settings admin page.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function initialize_option_settings() {
+	$args = [
+		'type'              => 'integer',
+		'group'             => 'extend-give-wp-options',
+		'description'       => 'The image ID for the donation form featured image.',
+		'sanitize_callback' => __NAMESPACE__ . '\sanitize_options',
```

Comment:
As this is an integer, you can eliminate the custom callback and use `absint`. Less code for you to manage. Here's a [link to codex.](https://codex.wordpress.org/Function_Reference/absint)

Reply:
Good to know. 

**Note:** Ultimately, Tonya removed the entire `$args` parameter from `register_settings()` as it wasn't needed to process the option form.